cypress.config.ts

```diff
@@ -10,10 +10,12 @@ export default defineConfig({
         pass: "Dog8code",
         git_user: "",
         git_password: "",
-        jira_url: "",
-        jira_key: "",
-        jira_email: "",
-        jira_token: "",
+        jira_stage_datacenter_url: "https://issues.stage.redhat.com",
+        jira_stage_datacenter_key: "ODIzNDc2OTk0NDg2OjqhAqjzbsoQ4of8jPGmAPQBgE+X",
```

Comment:
For clarity, could you please replace this with jira_stage_datacenter_bearer_token ?

Reply:
I can, but I guess it is clear enough already and I didn't want to make it too long

Reply:
jira_stage_bearer_token is fine